js/output/pjs/pjs-code-injector.js

```diff
@@ -156,13 +156,46 @@ class PJSCodeInjector {
 
             playSound: (sound) => {
                 if (sound && sound.audio && sound.audio.play) {
+					if( sound.audio.fadeOutInterval ) {
+						sound.audio.fadeOutStopInterval();                                    
+					}
+
                     sound.audio.currentTime = 0;
                     sound.audio.play();
                 } else {
                     throw {message: i18n._("No sound file provided.")};
                 }
             },
 
+			stopSound: (sound, fadeOut) => {
+				if (sound && sound.audio && sound.audio.volume) {
+					if( !fadeOut || fadeOut < 0.020 ) {
+						sound.audio.pause();
+					} else {
+						var volume = sound.audio.volume,
+							orgVolume = volume;
```

Comment:
When stopSound() is called during fadeout I (now ;-) stop the running interval when it is called without fade option and just return and do nothing when it is called with fade option. It would be possible to let it fade at the second (or consecutive calls) from whereever the volume is by then, but this is a higher complexity level and I am not sure if there are use cases for it.

What do you mean with default volume of the audio tag? I only know the volume property and found nothing in stackoverflow.